mlir/utils/jenkins/Jenkinsfile

```diff
@@ -225,6 +225,9 @@ void check_randomE2ETests(String codepath) {
               -DLLVM_LIT_ARGS='-v --time-tests ${ limit_lit_workers ? '-j 8' : ' ' }'
               -DCMAKE_EXPORT_COMPILE_COMMANDS=1
              """)
+    timeout(time: 60, activity: true, unit: 'MINUTES') {
+        sh "cd build; ninja check-rocmlir || ${codepath} == 'navi3x'"
```

Comment:
I don't think this is valid shell syntax. 

Also, it's wort checking (though it might not be possible) if there's a way to suppress failures from within a Jenkins pipeline.